### PR TITLE
Final submission updates

### PR DESCRIPTION
**Pull request by eddieh-xlnx**

### CODE REVIEW

File: docs/final_submission.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+* ``--workdir `pwd`/workdir ``provides a working directory for `/tmp`, etc. within container
+* ``--home `pwd`/home `` maps the present working directory as `/home` inside container
+* `--bind /tools` mounts the host system's `/tools` directory into the container, allowing it to
+access the host Vivado installation.
+* `--rocm --bind /etc/OpenCL` configures [GPU Access](#gpu-access)
+
+The remainder of the Apptainer command line simply runs the default make target from inside the
+container.
+
+Finally, in order to aid in development the Makefile target:
```
Comment (clavin-xlnx) [inline docs/final_submission.md:68]:
```suggestion
Finally, in order to aid in development of the Makefile target:
```

File: docs/final_submission.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
+# Final Submission
+
+## Key Details
+
+* Final submission process/requirements are largely the same as for the [alpha submission](alpha_submission.html),
+  but will be more strictly enforced.
```
Comment (clavin-xlnx) [inline docs/final_submission.md:6]: Minor nit, the first bullet ends with a period, all others do not

File: docs/final_submission.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+Contestants are required to submit a clone of the contest
+repository which has been modified to run their router in Apptainer.
+Specifically, organizers must be able to run the submitted router by calling only
+the `make run-container` target. By default, in a fresh checkout of the contest
+repository, this target will run `rwroute` in an Apptainer container.
+Thus in addition to their router contestants must supply a custom `*.def` file
+in the `final_submission` directory, as well as a Makefile that has been
+modified to run their router by default. To set the default router in the
+Makefile contestants must change the value of the `ROUTER` variable from
```
Comment (clavin-xlnx) [inline docs/final_submission.md:119]:
```suggestion
Makefile, contestants must change the value of the `ROUTER` variable from
```

**Review by clavin-xlnx — APPROVED** [review]

File: docs/final_submission.md (hunk 2 above)
Comment (eddieh-xlnx) [inline docs/final_submission.md:6]:
```suggestion
  but will be more strictly enforced
```

**Pull request merged**